pkg/cloud/amazon/eks/interface.go

```diff
@@ -0,0 +1,35 @@
+package eks
+
+import (
+	"github.com/aws/aws-sdk-go/service/eks"
+	"github.com/jenkins-x/jx/pkg/cluster"
+)
+
+// EKSer is an interface that abstracts the use of the EKS API
+//go:generate pegomock generate github.com/jenkins-x/jx/pkg/cloud/amazon/eks EKSer -o mocks/ekserMock.go
+type EKSer interface {
```

Comment:
Just _EKS_?

Reply:
What do you mean?

Reply:
_EKS_ instead of _EKSer_. Same as with the other interface. Never mind.